EpsilonAuxBMS/AuxBMSTests.adoc

```diff
@@ -14,89 +14,115 @@ Elapsed 1000ms{
 }
 ----
 
-== CAN Values to Test
+== Test Cases
 [options="header"]
 |===
-| Input Type | CAN_ADDRESS | DATA_INDEX | TIME_FRAME
-| PrechargeState |  0x721  | 0 |  100ms 
-| AuxVoltage |  0x721 |  0 |  100ms |
+| Test | CAN_ADDRESS | DATA_INDEX | TIME_FRAME
+| Precharge State |  0x721  |  |  100ms 
+| Aux Voltage |  0x721 |  3-7 |  100ms 
+| Charge Contactor State |  0x721  | 0-2 |  100ms 
+| Contactor State |  0x721  | 0 |  100ms 
+| All Contactor On |  0x721 |  0-2 |  100ms 
+| Discharge Off |  0x721  | 0-2 |  100ms 
+| Safety |  0x721 |  0-2 |  100ms 
+| Allow Charge |  0x721  | 9 |  100ms 
+| Strobe BMS Light |  0x721 | 8 |  100ms |
 |===
 
-== Precharge Test
+== Precharge State Test
 ----
-Send (PRECHARGE_CAN_MESSAGE(0x721)[DATA_INDEX])
-Within 100ms {
-    Expect (PRECHARGE_STATUS_CAN_MESSAGE(0x721)[DATA_INDEX])
+Send (PRECHARGE_CAN_MESSAGE(CAN_ADDRESS))
+	Elapsed TIME_FRAME{
+		Expect (PRECHARGE_CAN_MESSAGE(CAN_ADDRESS))
 }
 ----
 
-== AuxVoltage Test
+== Aux Voltage Test
 ----
-Every 100ms{
-    Expect (CURRENT_VOLTAGE_AUX_BATTERY_CAN_MESSAGE(0x721)[DATA_INDEX])
+//Start Test
+Elapsed TIME_FRAME{
+    Expect (VOLTAGE_CAN_MESSAGE(CAN_ADDRESS)[DATA_INDEX])
 }
 ----
 
-== Contactor Charge Test
+== Charge Contactor Test
 ----
-Every 100ms{
-	Expect (CONTACTOR_CURRENT_CAN_MESSAGE(0x721)[DATA_INDEX])
+//Start Test
+Within TIME_FRAME{
+	Expect (CONTACTOR_CAN_MESSAGE(CAN_ADDRESS)[DATA_INDEX])
 }
 ----
 
 == Contactor State Test
 ----
-Every 100ms{
-	Expect (CONTACTOR_STATE_CAN_MESSAGE(0x721)[DATA_INDEX])
+//Start Test
+Within TIME_FRAME{
+	Expect (CONTACTOR_CAN_MESSAGE(CAN_ADDRESS)[DATA_INDEX])
 }
 ----
 
 == All Contactor On Test
 ----
-Charge input test
+//Start Test
 
-Wait 5s{
-    Expect (CHARGE_CAN_MESSAGE(0x721)[DATA_INDEX] == 001) 
-     COMMON LED ON
-      }
-Wait 5s{
-    Expect (CHARGE_CAN_MESSAGE(0x721)[DATA_INDEX] == 011) 
-    COMMON LED ON
-    CHARGE LED ON
-    }
-  Wait 5s{
-    Expect (CHARGE_CAN_MESSAGE(0x721)[DATA_INDEX] == 111) 
-    COMMON LED ON
-    CHARGE LED ON
-    DISCHARGE LED ON
-    }
-  Wait 5s{
-    Expect (CHARGE_CAN_MESSAGE(0x721)[DATA_INDEX] == 111) 
-    COMMON LED ON
-    CHARGE LED ON
-    DISCHARGE LED ON
-    }
+Within 5s{
+    Expect (CONTACTOR_CAN_MESSAGE(CAN_ADDRESS)[DATA_INDEX] == 0b001) 
+        COMMON LED ON
+}
+Within 5s{
+    Expect (CONTACTOR_CAN_MESSAGE(CAN_ADDRESS)[DATA_INDEX] == 0b011) 
+        COMMON LED ON
+        CHARGE LED ON
+}
+Within 5s{
+    Expect (CONTACTOR_CAN_MESSAGE(CAN_ADDRESS)[DATA_INDEX] == 0b111) 
+        COMMON LED ON
+        CHARGE LED ON
+        DISCHARGE LED ON
+}
+Within 5s{
+    Expect (CONTACTOR_CAN_MESSAGE(CAN_ADDRESS)[DATA_INDEX] == 0b111) 
+   	    COMMON LED ON
+   	    CHARGE LED ON
+   	    DISCHARGE LED ON
+}
```

Comment:
These times are too long. The contactors turn on within 1 second. Also, these times should be specified in your table as TIME_FRAMEs